website/docs/CloudServices.md

```diff
@@ -18,6 +18,39 @@ if (process.env.CI) {
 exports.config = config
 ```
 
+
+## Perfecto
+
+When using wdio with Perfecto, you need to create a security token for each user and add this in the capabilities structure (in addition to other capabilities), as follows:
+  
+exports.config = {
+  capabilities: [
+    {
+    .
+    .
+    .
+    
+    securityToken:"your security token"
+    .
+       },
+  ],
```

Comment:
Also, can we wrap this code with 3 backticks so it is properly formatted as code?

```suggestion
exports.config = {
  capabilities: [{
    // ...
    securityToken: "your security token"
  }],
```